src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -110,6 +114,10 @@ public static int Main(string[] args)
                     "Display warnings and errors only.",
                     CommandOptionType.NoValue);
 
+                var checkJsonOutput = c.Option("--check-trust-machine-readable",
```

Comment:
@DamianEdwards is this good? Last chance to change it 😆

Reply:
This is the option to enable the JSON output? Is there not a prior art we can use from the CLI already? @baronfel

Reply:
> `--format json` is what we've been doing for most of these. the 'default' format by comparison is `text`.

This option is already used for the certificate export operation. What about `--output-format` instead? I don't think reusing the same option is a good idea

Reply:
`output-format` is fine with me. Or if we're not going to have any other dev-certs operations support different output types anytime soon, we can just go with `--check-trust-machine-readable` for now. If that changes in the future and we want more operations to be able to specify output format, we can figure out the consistency story with `--format`.

Reply:
Yep let's not block on this given `--format` conflicts with the existing option for certificate export format.